### PR TITLE
1st round chapter 8

### PR DESCRIPTION
**Pull request by wangkuiyi**

### CODE REVIEW

File: recommender_system/README.en.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+- Content-based recommendation[[1](#reference)]. This approach infers feature vectors that represent products from their descriptions.  It also infers feature vectors that represent users' interests.  Then it measures the relevance of users and products by some distances between these feature vectors.
+
+- Hybrid approach[[2](#reference)]: This approach uses the content-based information to help address the cold start problem[[6](#reference)] in behavior-based approach.
+
+Among these options, collaborative filtering might be the most studied one.  Some of its variants include user-based[[3](#reference)], item-based [[4](#reference)], social network based[[5](#reference)], and model-based.
+
+This tutorial explains a deep learning based approach and how to implement it using PaddlePaddle.  We will train a model using a dataset that includes user information, movie information, and rates.  Once we train the model, we will be able to get a predicted rate given a pair of user and movie IDs.
```
Comment (helinwang) [inline recommender_system/README.en.md]: and rates -> and ratings
get a predicted rate -> get a predicted rating

File: recommender_system/README.en.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
+# Personalized Recommendation
+
+The source code of this tutorial is in [book/recommender_system](https://github.com/PaddlePaddle/book/tree/develop/recommender_system).
+
+## Background
+
+With the fast growth of e-commerce, online video, and online reading business, users have to rely on recommender systems to avoid manually browsing tremendous volume of choices.  Recommender systems understand users' interest by mining user behavior and other properties of users and products.
```
Comment (zchen0211) [inline recommender_system/README.en.md]: online videos,

File: recommender_system/README.en.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+YouTube is a video-sharing Web site with one of the largest user base in the world.  Its recommender system serves more than a billion users.  This system is composed of two major parts: candidate generation and ranking.  The former selects few hundreds of candidates from millions of videos, and the latter ranks and outputs the top 10.
+
+<p align="center">
+<img src="image/YouTube_Overview.en.png" width="70%" ><br/>
+Figure 1. YouTube recommender system overview.
+</p>
+
+#### Candidate Generation Network
+
+Youtube poses candiate generation as extreme multiclass classification where the input is a user and related information, and the classification labels are  all (millions of) videos.  The architecture of the model is as follows:
```
Comment (helinwang) [inline recommender_system/README.en.md]: Maybe: Youtube poses candiate generation as extreme multiclass classification where the input ... -> Youtube model candidate generation as a classification problem with a huge number of classes. The input ...

are  all -> are all

**Review by helinwang — APPROVED** [review]
LGTM, have 2 comments.

File: recommender_system/README.en.md (hunk 3 above)
Comment (wangkuiyi) [reply recommender_system/README.en.md]: Done

File: recommender_system/README.en.md (hunk 2 above)
Comment (wangkuiyi) [reply recommender_system/README.en.md]: Done

File: recommender_system/README.en.md (hunk 1 above)
Comment (wangkuiyi) [reply recommender_system/README.en.md]: Done

**Pull request merged**